src/translations/installer_it.ts

```diff
@@ -236,5 +236,9 @@
         <source>Lang_it</source>
         <translation>Italiano</translation>
     </message>
+    <message>
+        <source>Lang_pt</source>
+        <translation>Português</translation>
```

Comment:
```suggestion
        <translation>Portoghese</translation>
```
This is the Italian translation file, so this should be translated in Italian.